content/blogs/GSoC-2020-rapps.md

```diff
@@ -0,0 +1,50 @@
+---
+title:       "Rapps Enhancement - GSoC 2020"
+author:      "kernel.bin"
+date:        2020-05-30
+tags:        [ gsoc ]
+---
+
+Hello, I am He Yang (@kernel.bin) from China. This year, I signed up for GSoC, and I'm very glad to be given the chance to work with ReactOS team.
+
+my mentors for the project are Mark Jansen and Ioannis Adamopoulos, It's a great honor to have two mentors to guide me.
+
+My project is "Application manager RAPPS". RAPPS is used by almost everyone using ReactOS, thus making it essential to get RAPPS easy-to-use, powerful and bug-free.
+
+## How I meet ReactOS and GSoC
+
+*This part is not related to the work I'm going to do, just some story between me and ReactOS*
+
+Less than a year ago, I first heard about ReactOS from a friend of mine when I traveling with him.
+
+We try to install ReactOS on a VM to see what exactly is it. Unfortunately, it crashed when installing and a blue screen was shown. But anyway that's my first meet with ReactOS. I didn't realize that I would work on this project in the future so soon.
+
+Several months ago, I first heard about GSoC (Interestingly, still from the same friend). When I try to find a project correspond with my skill, ReactOS came into my view.
+
+It's great for me because having a chance to write a part of an OS is exciting and there aren't many projects that require Win32 skills. And then I'm here!
+
+## Details about the project
+
+RAPPS is a lightweight GUI utility that downloads and installs various popular apps and redistributable packages in Windows and ReactOS. Aiming at making manage software easier, especially on ReactOS.
+
+Currently, RAPPS gots dozens of bugs to be fixed, and some function is broken or not implemented. besides, I consider the UI of RAPPS is a little bit old-fashioned (this may differ from man to man). My aim is to fix bugs as much as I can and improve RAPPS's functionality and UI.
```

Comment:
```suggestion
Currently, RAPPS has dozens of bugs to be fixed, and some function are broken or not implemented.
Besides, I consider the UI of RAPPS is a little bit old-fashioned (this may differ from man to man).
My aim is to fix bugs as much as I can and improve RAPPS's functionality and UI.
```